pubspec.lock

```diff
@@ -229,12 +229,26 @@ packages:
     source: hosted
     version: "1.8.0-nullsafety.1"
   pedantic:
-    dependency: "direct dev"
```

Comment:
Might be that `"direct dev"` is more suitable for this package, as we are directly using it.